src/pack/stream.ts

```diff
@@ -40,7 +40,8 @@ export async function packToStream ({ input, writable, blockstore: userBlockstor
       maxChildrenPerNode: maxChildrenPerNode || unixfsImporterOptionsDefault.maxChildrenPerNode,
       wrapWithDirectory: wrapWithDirectory === false ? false : unixfsImporterOptionsDefault.wrapWithDirectory,
       rawLeaves: rawLeaves == null ? unixfsImporterOptionsDefault.rawLeaves : rawLeaves
-    })
+    }),
+    (source: any) => printCarContent(source, verbose)
```

Comment:
Can we call this printUnixFsContent instead? It is more what it is than car content

Reply:
Also thinking this should not be part of the lib, but part of the CLI. So, what do you think on receiving an optional handler as parameter to `packToStream` that would be an async iterator of verbose option?

Than CLI can actually provide the printUnixFsContent, and users can also provide any other function they care, being cautious that in the end it must yield what it received.